go-controller/pkg/ovn/gateway_cleanup.go

```diff
@@ -26,7 +28,7 @@ func (oc *DefaultNetworkController) gatewayCleanup(nodeName string) error {
 	var nextHops []net.IP
 
 	gwIPAddrs, err := util.GetLRPAddrs(oc.nbClient, types.GWRouterToJoinSwitchPrefix+gatewayRouter)
-	if err != nil {
+	if err != nil && !errors.Is(err, libovsdbclient.ErrNotFound) {
```

Comment:
hmm... I'm also doing this on https://github.com/ovn-org/ovn-kubernetes/pull/3647 ... can we just merge that and rebase this?

Reply:
doesnt really matter who's goes in first, its the same change